sherpa/optmethods/tests/test_optmethods.py

```diff
@@ -39,13 +38,22 @@ def tst_all( self, name, fct, fmin, x0, xmin, xmax,
                  iprint=False ):
         self.tst( optfcts.neldermead, name + self.nm, fct, fmin,
                   x0, xmin, xmax, iprint=iprint )
-##         self.tst( stogo.stogo, name + '_stogo', fct, fmin,
-##                   x0, xmin, xmax, iprint=iprint )
         self.tst( optfcts.montecarlo,  name + self.mc , fct, fmin,
                   x0, xmin, xmax, iprint=iprint, maxfev=8192*len(x0) )
         self.tst( optfcts.lmdif, name + self.lm, fct, fmin,
                   x0, xmin, xmax, iprint=iprint )
 
+        #
+        # If the following lines are uncommented out, we get errors
+        # when running the optimization test
```

Comment:
minuit failed to solve a few (can't remember now how many but if I have to
guess it was 6, I can check if inquiring minds want to know) problems that
the other methods can

On Fri, Mar 25, 2016 at 8:33 AM, Doug Burke <notifications@github.com>
wrote:

> In sherpa/optmethods/tests/test_optmethods.py
> <https://github.com/sherpa/sherpa/pull/194#discussion_r57440321>:
>
> >          self.tst( optfcts.montecarlo,  name + self.mc , fct, fmin,
> >                    x0, xmin, xmax, iprint=iprint, maxfev=8192*len(x0) )
> >          self.tst( optfcts.lmdif, name + self.lm, fct, fmin,
> >                    x0, xmin, xmax, iprint=iprint )
> >
> > +        #
> > +        # If the following lines are uncommented out, we get errors
> > +        # when running the optimization test
>
> Are the errors because the iminuit test itself fails, or is it a more
> general error (e.g. unable to import the module or some other error)?
>
> —
> You are receiving this because you authored the thread.
> Reply to this email directly or view it on GitHub
> <https://github.com/sherpa/sherpa/pull/194/files/3d8653dfe45d619b1a8a514d7dda345dd38b655f#r57440321>
>